### DISCUSSION TITLE
Project unmaintained?

### DISCUSSION THREAD

**Discussion by Tom-H-L** [spaceuptech/space-cloud#1616] [General]
Hello all!

Looking at the Insights, I wonder if this project dead & unmaintained?

Best

**Comment by YourTechBud** [comment]
The project still has active development. Work is happening on the `v0.22.0` branch. There is some enterprise adoption of the project as well who are sponsoring to keep the project going.

The development efforts are at a less than ideal pace rn. Hopefully this will pick up some steam in the coming weeks.
